.circleci/config.yml

```diff
@@ -71,6 +71,10 @@ setup_minikube: &setup_minikube
       true
     )
 
+    # This installs nginx ingress controller
```

Comment:
```suggestion
    # This installs the nginx ingress controller
```

Reply:
FYI this has been migrated to `module-ci`: https://github.com/gruntwork-io/module-ci/pull/91